soundbay/data.py

```diff
@@ -288,6 +290,25 @@ def __call__(self, sample):
         return (sample - sample.min()) / (sample.max() - sample.min())
 
 
+class MinFreqSpectrogram:
+    """Cut the spectrogram frequency axis to make it start from min_freq"""
+    def __init__(self, min_freq_spectrogram, sample_rate):
```

Comment:
need to add type check in function definition.
`sample_rate` is integer , `min_freq_spectrogram` is `Tensor`(I Think?)